tst/testrandom.g

```diff
@@ -1,34 +1,38 @@
-randomTest := function(collection, method, checkin...)
-    local test1, test2, test3, test4, test5, test6, localgen, checkmethod;
-    if Length(checkin) = 0 then
-        checkmethod := \in;
-    else
-        checkmethod := checkin[1];
-    fi;
+# Perform a variety of tests on Random generators.
+# filter: The type of random source we are testing
+# globalobj: The name of the global instance of this iterator
```

Comment:
"this iterator" ? Huh?

Also, why is this named `globalobj`, and in how far does the object passed in have to be "global" ? If I create such an instance on the fly, and also create gmethod on the fly, using this, it would be fine, too?

How about: 
> globalrs: an instance of a random source in <filter> (typically a global instance thereof, like <GlobalMersenneTwister>, to be used by gmethod.